tiflash/use-tiflash.md

```diff
@@ -503,19 +503,27 @@ To access partitioned tables in the MPP mode, you need to enable [dynamic prunin
 
 > **Warning:**
 >
-> - Currently, dynamic pruning mode for partitioned tables is an experimental feature and is not recommended for production environments.
->
-> - Do not enable dynamic pruning mode when a partitioned table contains columns of the `time` type. Otherwise, TiFlash crashes when a query selects a column of the `time` type.
+> - Do not enable dynamic pruning mode when a partitioned table contains columns of the `time` type. Otherwise, TiFlash crashes when a query selects a column of the `time` type. (TODO: Check if this is still correct?!)
```

Comment:
```suggestion
```

Reply:
Check and did not find any error when doing a simple test. Removed.